cfgov/v1/templates/v1/images_report.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'wagtailadmin/reports/base_report.html' %}
+{% load i18n wagtailadmin_tags %}
+
+{% block actions %}
+    {% if view.list_export %}
+        <div class="actionbutton">
+            <a href="{{ view.csv_export_url }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a>
+        </div>
+    {% endif %}
+{% endblock %}
```

Comment:
By default Wagtail offers both XLSX and CSV downloads for reports. Digging through history I see we originally removed the XLSX option from the page metadata report in #6060 (see internal platform#3881 for background). This same template code was copied for the document report in #6900, and again here.

1. Do we need/want to eliminate the XLSX option for the document and image reports?
2. If so, would it make sense to implement something like a `v1/reports/report_no_xlsx_base.html` base template that all of these report templates inherit from, to avoid the duplication?

Reply:
> If so, would it make sense to implement something like a `v1/reports/report_no_xlsx_base.html` base template that all of these report templates inherit from, to avoid the duplication?

@chosak, yeah, it's probably a good idea at this point to make a no-Excel base template and have all the other custom reports inherit from it.

We eliminated the Excel option originally because the page metadata report was so big it would crash the Excel export in production. I'm not sure if Wagtail has improved how it exports since then or what size is big enough to cause issues, though. In general the Excel export doesn't offer much over the CSV, so I'd be fine making a base template with just the CSV export and using that base template for all of our custom reports.

Reply:
Agree that I don't see much need for an XLSX option. I can work on building a `v1/reports/report_no_xlsx_base.html` and adding it to this commit (including corresponding edits to other pages as needed).

Reply:
FYI the page metadata report inherits from a different base report, so the change to do this that I just committed only affects the documents report and this new image report.